.dependabot/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 1
+
+update_configs:
+  - package_manager: "javascript"
+    directory: "/packages/stark-build"
+    update_schedule: "live"
+    automerged_updates:
+      - match:
+          dependency_type: "all"
+          update_type: "in_range"
+    ignored_updates:
+      - match:
+          dependency_name: "@angular/*"
+          version_requirement: ">=8"
+      - match:
+          dependency_name: "@angular-builders/*"
+          version_requirement: ">=8"
+      - match:
+          dependency_name: "@angular-devkit/build-angular"
+          version_requirement: ">=0.800"
+
+  - package_manager: "javascript"
+    directory: "/packages/stark-core"
+    update_schedule: "live"
+    automerged_updates:
+      - match:
+          dependency_type: "all"
+          update_type: "in_range"
+    ignored_updates:
+      - match:
+          dependency_name: "@uirouter/angular"
+          version_requirement: ">=6"
+      - match:
+          dependency_name: "@types/node"
```

Comment:
I'm wondering if we should also indicate the versions of `@types/node` we don't want to upgrade to (I think it would be `>=11` right?). What do you think?

Reply:
You're right, it could be a good idea. Let's do it to maintain the types up to date :blush: